### PR TITLE
docs: update setup with poll finalization

### PR DESCRIPTION
**Pull request by crisgarner**
Update docs to reflect poll finalization

### CODE REVIEW

File: docs/01_setup.md
Comment (ctrlc03) [inline docs/01_setup.md:166]: something to keep in mind is that for 1.2.2 you cannot use the same maci contract for more than one poll, as if one user signups after the first poll has been finalized, the second poll won't be able to be finalized (this works now in the latest dev)

File: docs/01_setup.md
Comment (ctrlc03) [inline docs/01_setup.md:80]: please can you change the part after the comma to: "as this version's circuit are audited and have zKeys which have undergone a trusted setup".  The contracts have some small changes (removal of subsidy and Mode for NonQv vs Qv) since the audit, so worth being accurate with wording here.

**Review by ctrlc03 — CHANGES_REQUESTED** [review]
Thanks, I left a couple of comments

File: docs/01_setup.md
Comment (kittybest) [reply docs/01_setup.md:166]: also wanna leave a question here: I think we wanna let the recent qf rounds (eth mexico?) to use v1.2.2, then how should we update the document? to a certain branch? since the latest branch is based on the latest version of maci.

File: docs/01_setup.md
Comment (kittybest) [inline docs/01_setup.md:174]: I think the wasm and zkey files are still needed?

File: docs/01_setup.md
Comment (crisgarner) [reply docs/01_setup.md:166]: You need to finalize it before starting a new poll or you can't use the same MACI contract?

File: docs/01_setup.md
Comment (crisgarner) [reply docs/01_setup.md:174]: yes! I made an update for the tasks so it can grab it directly from the config file, will make a PR after I finish testing

**Comment by kittybest** [comment]
I think this is blocked by: https://github.com/privacy-scaling-explorations/maci-rpgf/pull/117

File: docs/01_setup.md
Comment (ctrlc03) [reply docs/01_setup.md:166]: short story, you can't. if another user signs up after the trees are merged, then there is a skew in the state index which causes a mismatch with the local code

**Comment by kittybest** [comment]
I think the default network for maci-rpgf should be `optimismSepolia`?
https://github.com/privacy-scaling-explorations/maci-rpgf/blob/afce8c433367fdb8a32186b59aa108a62e50a4e2/docs/01_setup.md?plain=1#L38

File: docs/01_setup.md
Comment (ctrlc03) [inline docs/01_setup.md:93]: in v1.2.2 you need to download the zkeys with: `pnpm download:test-zkeys`

**Review by ctrlc03 — CHANGES_REQUESTED** [review]
Left one more comment on the test zkeys

**Review by ctrlc03 — APPROVED** [review]

File: docs/01_setup.md
Comment (kittybest) [inline docs/01_setup.md:93]: do we need to specify this?
```
pnpm download:test-zkeys-1-3
```

Comment (ctrlc03) [reply docs/01_setup.md:93]: https://github.com/privacy-scaling-explorations/maci-rpgf/pull/110#discussion_r1609485727

Comment (ctrlc03) [reply docs/01_setup.md:93]: also, gonna open a pr soon to remove 1-3/1-2 from the download command and have only one

Comment (kittybest) [reply docs/01_setup.md:93]: sorry missed that, thanks!

**Pull request merged**
